src/edit.c

```diff
@@ -3589,6 +3590,36 @@ ins_compl_active(void)
     return compl_started;
 }
 
+// Return Insert completion mode name string
```

Comment:
```c
/*
 * Return Insert completion mode name string
 */
```

Why do you get worse?
It has already been pointed out by Bram.
See the document.

```vim
 :h style-examples
/function_name
```